eo-runtime/src/main/java/org/eolang/JulToSlf4jExtension.java

```diff
@@ -0,0 +1,26 @@
+package org.eolang;
```

Comment:
@marceloamadeu I suggest moving this class to `src/test/java`

Reply:
@yegor256 done...